docs/faq/DAGMan/bagman-exit-at-percent-done.rst

```diff
@@ -0,0 +1,149 @@
+Example: DAGMan Exit Upon Percent Completed
+===========================================
```

Comment:
"Exit" seems a little overloaded here.  Can we title this

`DAGMan completes when some percentage of nodes succeed` or something like that?